src/jit/importer.cpp

```diff
@@ -7664,6 +7676,22 @@ var_types Compiler::impImportCall(OPCODE                  opcode,
             }
             else
             {
+                if (fatPointerCandidate)
+                {
+                    // fatPointer candidates should be in statements of the formL call() or var = call().
```

Comment:
` Such form allows to find statements with fat calls without walking through whole trees and removes problems with cutting trees.`
I will add this to the comment.